src/sentry/rules/history/preview.py

```diff
@@ -387,68 +387,92 @@ def apply_frequency_conditions(
-                        bucket_time,
+                        dataset_map[group],
+                        conditions[0].get_preview_aggregate(),
                     )
-                    try:
-                        if conditions[0].passes_activity_frequency(activity, buckets):
-                            activities.append(activity)
-                    except NotImplementedError:
-                        raise PreviewException
-
-            passes = [True] * len(activities)
-            for condition in conditions if has_issue_state_condition else conditions[1:]:
-                for i, activity in enumerate(activities):
-                    try:
-                        if passes[i] and not condition.passes_activity_frequency(activity, buckets):
-                            passes[i] = False
-                    except NotImplementedError:
-                        raise PreviewException
-
-            filtered_activity[group] = [activities[i] for i in range(len(activities)) if passes[i]]
-
-        return filtered_activity
+                except NotImplementedError:
+                    raise PreviewException
+                skip_first = False
+                if not has_issue_state_condition and not init_activities_from_freq_cond:
+                    # If there are no issue state change conditions, then we won't have any initial activities
+                    # to base our frequency condition queries off of. Instead, we take the first frequency condition and
+                    # create the initial activities from that
+                    init_activities_from_freq_cond = skip_first = True
+                    for bucket_time in buckets.keys():
+                        activity = ConditionActivity(
+                            group,
+                            ConditionActivityType.FREQUENCY_CONDITION,
+                            bucket_time,
+                        )
+                        try:
+                            if conditions[0].passes_activity_frequency(activity, buckets):
+                                activities.append(activity)
+                        except NotImplementedError:
+                            raise PreviewException
+
+                passes = [True] * len(activities)
+                for condition in conditions[1:] if skip_first else conditions:
+                    for i, activity in enumerate(activities):
+                        try:
+                            if passes[i] and not condition.passes_activity_frequency(
+                                activity, buckets
+                            ):
+                                passes[i] = False
+                        except NotImplementedError:
+                            raise PreviewException
+
+                filtered_activity[group] = [
+                    activities[i] for i in range(len(activities)) if passes[i]
+                ]
+
+            return filtered_activity
```

Comment:
is this indented correctly?

Reply:
and, if not, can we add a test that will catch this?

Reply:
whoops this and the `passes` array are definitely not in the right scope, I'll change it and add a test